javascript/grid-ui/src/setupTests.ts

```diff
@@ -0,0 +1,3 @@
+import { configure } from 'enzyme';
```

Comment:
This file should probably be a `.tsx` file so we don't get that weird issue we had before